Makefile

```diff
@@ -49,22 +50,32 @@ patch-post:
 clean:
 	rm -rf $(PACKAGE_PREFIX)
 
-gen:
+mark-paginated:
+	rm -rf ${SPEC_MARK_PAGINATED_DIR}
+	cp -r ${SPEC_PATCHED_DIR} ${SPEC_MARK_PAGINATED_DIR}
+	for operation_spec in $(shell set -x; find ${SPEC_MARK_PAGINATED_DIR}/paths -name '*.yaml' | sort -n); do \
+		script/mark_paginated.py $$operation_spec; \
+	done
+    
+
+gen: validate
 	${OPENAPI_GENERATOR} generate -g go \
 		--package-name ${PACKAGE_MAJOR} \
 		--http-user-agent "${GIT_REPO}/${PACKAGE_VERSION}" \
 		-p packageVersion=${PACKAGE_VERSION} \
 		-p isGoSubmodule=true \
 		-p disallowAdditionalPropertiesIfNotPresent=false \
+		-c /local/config.yaml \
+		--global-property apis \
```

Comment:
I was curious about the options here: https://openapi-generator.tech/docs/customization#selective-generation

Reply:
`--global-property apis` shouldn't be there, it's been left there by accident.